website_product_show_uom/README.rst

```diff
@@ -0,0 +1,84 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
```

Comment:
Please include link like in https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst